proto/internal/temporal/server/api/replication/v1/message.proto

```diff
@@ -194,3 +195,18 @@ message SyncHSMAttributes {
     temporal.server.api.history.v1.VersionHistory version_history = 4;
     temporal.server.api.persistence.v1.StateMachineNode state_machine_node = 5;
 }
+
+message BackfillHistoryTaskAttributes {
+    string namespace_id = 1;
+    string workflow_id = 2;
+    string run_id = 3;
+
+    temporal.server.api.persistence.v1.VersionedTransition versioned_transition = 4;
```

Comment:
@xwduan have added a top level field for versioned_transition under ReplicationTask, guess we can just use that.

Reply:
removed this one.